### PR TITLE
UOE-7051: Fix pubmatic aliases buyid targeting key

### PR DESCRIPTION
**Pull request by pm-nilesh-chate**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [x] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [x] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

File: adapters/bidder.go [hunk 1]
```diff
@@ -125,6 +125,8 @@ type RequestData struct {
 	Uri     string
 	Body    []byte
 	Headers http.Header
+
+	BidderName openrtb_ext.BidderName
```
Comment (pm-viral-vala) [inline adapters/bidder.go:129]: json annotate "-"

**Review by pm-viral-vala — COMMENTED** [review]
Please create master branch PR

File: adapters/bidder.go (hunk 1 above)
Comment (pm-nilesh-chate) [reply adapters/bidder.go]: done

**Comment by pm-nilesh-chate** [comment]
> Please create master branch PR

done

**Review by pm-viral-vala — APPROVED** [review]

**Pull request merged**